File: java/htmx_router/src/main/java/com/ace/htmxrouter/PageService.java
package com.ace.htmxrouter;

import org.springframework.stereotype.Service;
import org.springframework.ui.Model;

import java.util.Date;
import java.util.Map;

@Service
public class PageService {

    public void stuff(){}

    public String openPage(Page page, Model model, boolean fragment, Map<String, String> allRequestParams) {
        model.addAttribute("time", new Date());
        model.addAttribute("name","coucou");
        return page.fragmentPath;
    }
}

File: java/picothymeleaf/src/main/kotlin/com/ace/picothymeleaf/PageService.kt
package com.ace.picothymeleaf

import org.springframework.stereotype.Service
import org.springframework.ui.Model
import java.util.*

@Service
class PageService {

fun loadMainDate(model: Model){
    model.addAttribute("time", Date())
    model.addAttribute("name","coucou")
}
}
